frontend/public/locales/zh/custom-resource-definition.json

```diff
@@ -0,0 +1,20 @@
+{
+  "View instances": "查看实例",
+  "true": "true",
+  "false": "false",
+  "CRD versions": "CRD 版本",
+  "Name": "名称",
+  "Served": "服务的",
+  "Storage": "存储",
+  "{{resource}} details": "{{resource}}详情",
+  "Established": "建立",
+  "Group": "Group",
```

Comment:
"Group" could be translated into Chinese.

Reply:
This has been updated.